integration-tests/test_display_name_option.py

```diff
@@ -133,3 +133,59 @@ def test_invalid_display_name(invalid_display_name, insights_client):
     assert (
         origin_display_name == record["display_name"]
     ), "display-name should remain unchanged when new display-name is rejected"
+
+
+def test_display_name_disable_autoconfig_and_autoupdate(insights_client, test_config):
+    """
+    This is a satellite bug(https://issues.redhat.com/browse/ESSNTL-2230),
+    determine test this case for both satellite and production
+    environment after discussion with Archana.
```

Comment:
Also you can remove names from docstring. and details can be added in the Jira card that is tracking this automation task.